components/onboarding-modal/OnboardingModal.js

```diff
@@ -25,6 +25,39 @@ const StepsProgressBox = styled(Box)`
   }
 `;
 
+const ResponsiveModal = styled(Modal)`
+  @media screen and (max-width: 40em) {
+    transform: translate(0%, 0%);
+    position: fixed;
+    top: 70px;
+    left: 0px;
+    height: calc(100vh - 70px);
+    background: white;
+    max-width: 100%;
+    border: none;
+    border-radius: 0;
+    padding: 0px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+`;
+
+const ResponsiveModalHeader = styled(ModalHeader)`
+  @media screen and (max-width: 40em) {
```

Comment:
Why these empty blocks?

Reply:
Because it's a draft PR. I struggled a lot yesterday and I put up my code to ask for some help (i,e, the specific question I asked above), I didn't really expect you'd comment on this. It's blocks of code I'm going to put CSS in. I didn't quite get to them yesterday.

Reply:
You asked me for feedback about this PR and because we were discussing how to style the different part of the Modal, I thought that was what you wanted me to look at your implementation for this. I also thought that these empty blocks may have served a particular purpose, which is why I asked.

Maybe we should avoid technical discussions on PRs while they're not ready for review and rather have them on Slack to avoid these kind of confusion in the future.